data/coalition/coalition outfits.txt

```diff
@@ -430,7 +430,8 @@ outfit "Anti-Materiel Gun"
 outfit "Coalition License"
 	category "Licenses"
 	thumbnail "outfit/coalition license"
-	description "Completed a lot of Coalition jobs and received permission to purchase civilian technology and ships."
+	description "A digital chip in the form of a stylized badge, this document signifies the permission of the owner to browse, employ and deal in transactions involving civilian Coalition technology."
+	description "	There is a history of controversy surrounding this license, as throughout the millennia several Heliarch laws have been in a back and forth regarding the conditions that need to be met for someone to be issued one, starting with the third Civil Goods and Fares Act, just a few decades after the War of Independence, all the way to the relatively recent twenty-sixth addendum to the second article of the Ninth Rectification on Commerce and Traffic, which two hundred years ago made the acquisition requirements for such a license much more flexible."
```

Comment:
For some reason GitHub didn't let me suggest changes above.
```suggestion
	description "	There is a history of controversy surrounding this license, as throughout the millennia several Heliarch laws have been in a back and forth regarding the conditions that need to be met for someone to obtain a license. This argument started with the third Civil Goods and Fares Act, created just a few decades after the War of Independence, and has continued all the way to the relatively recent twenty-sixth addendum to the second article of the Ninth Rectification on Commerce and Traffic, which was passed two hundred years ago and made the acquisition requirements for such a license much more flexible."
```
The second part probably still needs a little breakup, but I have no idea where to put it.